internal/cli/args.go

```diff
@@ -109,6 +116,9 @@ func parseEditResult(b []byte) ([]string, error) {
 }
 
 func (c *argsCmd) run(ctx context.Context, args []string) error {
+	logLvl := logger.Get(ctx).Level()
+	ctx = logger.WithLogger(ctx, logger.NewLogger(logLvl, c.streams.ErrOut))
```

Comment:
```suggestion
	level := logger.Get(ctx).Level()
	ctx = logger.WithLogger(ctx, logger.NewLogger(level, c.streams.ErrOut))
```

Reply:
I just got rid of the variable entirely, instead